scripts/run_olserver.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# This script git pulls master for openlibrary, infogami, and olsystem
+# and then starts the Docker service specified by $SERVICE
+
+cd /opt/olsystem
+sudo git pull origin master
+cd /opt/openlibrary
+sudo git pull origin master
+cd /opt/openlibrary/vendor/infogami
+sudo git pull origin master
```

Comment:
Won't need sudo any more for these :) 

```suggestion
git pull origin master
cd /opt/openlibrary
git pull origin master
cd /opt/openlibrary/vendor/infogami
git pull origin master
```